### PR TITLE
feature/SOF-4481 - Add Hidden Markov Model example

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: examples/assets/Molecules_Dataset_Collection/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (timurbazhirov) [inline examples/assets/Molecules_Dataset_Collection/LICENSE:1]: Not sure this file is needed - there's a repo-wide license already

Comment (AcylSilane) [reply examples/assets/Molecules_Dataset_Collection/LICENSE:1]: Adjusting the approach to fork the repo and git clone it

File: examples/job/SmilesMarkov.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python
```
Comment (timurbazhirov) [inline examples/job/SmilesMarkov.py:1]: This iPyNb file needs a better name

Comment (AcylSilane) [reply examples/job/SmilesMarkov.py]: Renamed

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**